src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resArray = [];
```

Comment:
```suggestion
  const resArray = [];
```

Please choose more meaningful names, it is the array of history, also avoid using data types in names